digitalidentity/service_test.go

```diff
@@ -90,7 +105,7 @@ func TestCreateShareURL_Unsuccessful_400(t *testing.T) {
 	assert.Check(t, !temporary || !tempError.Temporary())
 }
 
-func createShareUrlWithErrorResponse(statusCode int, responseBody string) (share ShareURL, err error) {
+func createShareUrlWithErrorResponse(statusCode int, responseBody string) (share SessionResult, err error) {
```

Comment:
can't comment above
Let's add some Create/Fetch specific errors so we can test that mapping that was missing.